archive_query_log/orm.py

```diff
@@ -190,6 +190,18 @@ class Snippet(SnippetId):
     text: str | None = Text()
 
 
+class DirectAnswerId(InnerDocument):
+    id: str = Keyword()
+    rank: int = Integer()
```

Comment:
Does a direct answer really have an interpretable "rank" (i.e., are the direct answers ordered in some way)?